frontend/src/p1/FileUploadPage.js

```diff
@@ -1,5 +1,8 @@
 /**@jsx jsx */
```

Comment:
Did you base this branch another branch (not on master)? not sure why this file is being modified here.